jobs/config.json

```diff
@@ -8804,12 +8804,12 @@
       "--kubernetes-anywhere-kubernetes-version=ci/latest",
       "--kubernetes-anywhere-proxy-mode=ipvs",
       "--provider=kubernetes-anywhere",
-      "--test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\] --minStartupPods=8",
+      "--test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\]|\\[sig-network\\]\\sServices --minStartupPods=8",
```

Comment:
IIUC, [sig-network] services related conformance test cases < conformance test cases? Why do you say "cover more test case"?

Reply:
Actually I used `|` before `[sig-network] Services`, it will run all `[conformance`]` test cases and all cases in `[sig-network] Services`.

Reply:
> Actually I used | before [sig-network] Services

ACK.